internal/tracing/manifests/otelcol/config.go

```diff
@@ -0,0 +1,15 @@
+package otelcol
+
+import (
+	"github.com/ViaQ/logerr/v2/kverrors"
+	"gopkg.in/yaml.v2"
```

Comment:
```suggestion
	"gopkg.in/yaml.v3"
```
Thats the version the collector and operator uses.